src/api/authorData.js

```diff
@@ -38,18 +38,18 @@ const getSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
 });
 
 // FIXME: DELETE AUTHOR
-const deleteSingleAuthor = (firebaseKey, uid) => new Promise((resolve, reject) => {
+const deleteSingleAuthor = (firebaseKey) => new Promise((resolve, reject) => {
```

Comment:
```suggestion 
const deleteSingleAuthor = (firebaseKey, uid) => new Promise((resolve, reject) => {
```

Pass the uid when calling deleteSingleAuthor so you can call getAuthors and resolve the new author array.